lib/haml_lint/adapter.rb

```diff
@@ -21,7 +21,7 @@ def self.detect_class
       case version
       when '~> 4.0' then HamlLint::Adapter::Haml4
       when '~> 5.0', '~> 5.1', '~> 5.2' then HamlLint::Adapter::Haml5
-      when '~> 6.0', '~> 6.0.a', '~> 6.1' then HamlLint::Adapter::Haml6
+      when '~> 6.0', '~> 6.0.a', '~> 6.1', '~> 6.2' then HamlLint::Adapter::Haml6
```

Comment:
should we change `haml_version` to `haml_major_version` and go for `segments.first.to_i`?

```
      when 4 then HamlLint::Adapter::Haml4
      when 5 then HamlLint::Adapter::Haml5
      when 6 then HamlLint::Adapter::Haml6
```